Itaka scraper/scrape_links_to_offers.py

```diff
@@ -1,26 +1,26 @@
 from selenium.webdriver.common.by import By
-from offer_class import Offer
+from tour_class import Tour
 
 
 def scrape_all_inclusive_offers(driver):
     pages_to_check = 2
-    offers = []
+    tours = []
     for pg in range(1, pages_to_check + 1):
         website_link = "https://www.itaka.pl/all-inclusive/?page=" + str(pg)
         driver.get(website_link)
 
         load_whole_page(driver)  # scroll to the end of page
 
-        # find all offers on the page
-        offers_on_website = driver.find_elements(By.CLASS_NAME, "styles_c--secondary__93kD2")[1:-1]
+        # find all tours on the page
+        tours_on_website = driver.find_elements(By.CLASS_NAME, "styles_c--secondary__93kD2")[1:-1]
 
-        # save all offers
-        for offer in offers_on_website:
-            offer_web_link = offer.get_attribute('href')
-            new_offer = Offer(offer_web_link)
-            offers.append(new_offer)
+        # save all tours
+        for tour in tours_on_website:
+            tour_web_link = tour.get_attribute('href')
+            new_tour = tour(tour_web_link)
```

Comment:
I reckon this should be 
```suggestion
            new_tour = Tour(tour_web_link)
```

Reply:
That's right. I am fixing some other bugs now